packages/pagination/src/styled/Pagination/StyledPageBase.ts

```diff
@@ -111,15 +111,11 @@ export const StyledPage = styled.li.attrs({
     font-weight: ${props => props.theme.fontWeights.semibold};
   }
 
-  ${props => colorStyles(props)};
-
-  &${props => (props.theme.rtl ? ':last-of-type' : ':first-of-type')} {
-    margin-left: 0;
-  }
+  ${props => !(props as any).disabled && colorStyles(props)};
```

Comment:
Always call `colorStyles`. Add the following to the end of the `colorStyles` function:

```css
:disabled,
[aria-disabled='true'] {
  background-color: transparent;
  color: /* applicable color overrides go here */;
}
```

Otherwise the code is relying on UA color treatment – which isn't right.